tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md

```diff
@@ -0,0 +1,39 @@
+Test1
+```sh
+# Mountpoint:/test
```

Comment:
Btw. the restore technique needs special handling of `/elektra` (needs to be done first). It is basically the same as stash in #876.